backend/requirements.txt

```diff
@@ -1,5 +1,4 @@
 Django==2.2.10
-PyYAML==5.3
```

Comment:
PyYaml should not be removed as it is used, for instance, in `docker/start.py`

Reply:
This `start.py` is not part of the platform per se, it's more of an utils to help launch the platform locally. The requirements.txt is supposed the described the requirements of the platform itself since it is in the dockerfiles.

I think it's ok to remove it here but we should also create another requirements file for local uses (`start.py`, `populate.py` etc.)